docs/mixpanel.md

```diff
@@ -0,0 +1,33 @@
+### What is Mixpanel?
+It is an analytic tool that gets you insighst on how users interact with your digital product. These insights will help enable you to make decisions on how your users can get the best out of your digital productand and eventually retain them.
+
+Get an overview of the Mixpanel: https://developer.mixpanel.com/docs/what-is-mixpanel 
+
+### How to set it up?
+
+- On the Mixpanel dashboard, create a project and get the project token.
+
+- On your machine, navigate to your project directory and in the terminal, run the following command `npm install --save mixpanel-browser` or `yarn add mixpanel-browser` to install the package
+- After installing the package, open your text editor,
+    - In App.js, start using Mixpanel in your project:
```

Comment:
Can you permalink to the specific lines that someone would need to update? 

Probably this line for the `mixpanel.init`: https://github.com/brittanyjoiner15/eui-event-template/blob/ebd21d40848eadbde0db013d9738cb7e754fc2ec/src/App.js#L9

and this line for the import: https://github.com/brittanyjoiner15/eui-event-template/blob/ebd21d40848eadbde0db013d9738cb7e754fc2ec/src/App.js#L6

So maybe show that line specifically and give an example of how you would set it up. For instance, I might rework this paragraph to say: 


```
After installing the package, open your text editor and navigate to [`app.js`](https://github.com/brittanyjoiner15/eui-event-template/blob/main/src/App.js). 
 
Start by importing mixpanel. Add this line [below the other imports](https://github.com/brittanyjoiner15/eui-event-template/blob/ebd21d40848eadbde0db013d9738cb7e754fc2ec/src/App.js#L6): 
`import mixpanel from 'mixpanel-browser'`;  

Next, you'll need to initiate mixpanel. Add another line [at the beginning of the app function](https://github.com/brittanyjoiner15/eui-event-template/blob/ebd21d40848eadbde0db013d9738cb7e754fc2ec/src/App.js#L9) that initializes mixpanel). 
`mixpanel.init('YOUR_TOKEN', {debug: true});`

You can find your token via.... 

```